sql/moz-fx-data-shared-prod/fxci/dataset_metadata.yaml

```diff
@@ -0,0 +1,15 @@
+friendly_name: Fxci
+description: |-
+  Please provide a description for the dataset
```

Comment:
```suggestion
  Data related to the Firefox-CI Taskcluster instance
```

Reply:
Oops, I re-generated these and forgot to re-update the description.